packages/react-router/__tests__/useResolvedPath-test.tsx

```diff
@@ -277,12 +277,14 @@ describe("useResolvedPath", () => {
         {
           routes: (
             <Route path="foo">
-              <Route
-                path="*"
-                element={
-                  <Component desc='<Route path="/foo"><Route path="*" />' />
-                }
-              />
+              <Route>
+                <Route
+                  path="*"
+                  element={
+                    <Component desc='<Route path="/foo"><Route path="*" />' />
+                  }
+                  />
+              </Route>
```

Comment:
Would you mind laving the existing test as-is and added a new test for the pathless layout scenario?

Reply:
Done, unless you meant adding a separate test file, like `gh-issure-8127-test.tsx`.